great_expectations/profiler/domain_builder/simple_semantic_type_domain_builder.py

```diff
@@ -1,16 +1,20 @@
 from typing import Iterable, List, Optional, Union
 
-from great_expectations.execution_engine.execution_engine import MetricDomainTypes
+import great_expectations.exceptions as ge_exceptions
+from great_expectations.core.domain_types import MetricDomainTypes, SemanticDomainTypes
 from great_expectations.profiler.domain_builder.column_domain_builder import (
     ColumnDomainBuilder,
 )
-from great_expectations.profiler.domain_builder.domain import (
-    Domain,
-    SemanticDomainTypes,
-)
+from great_expectations.profiler.domain_builder.domain import Domain
 from great_expectations.validator.validator import MetricConfiguration, Validator
 
 
+# TODO: <Alex>ALEX -- This method seems to always return the same value ("IDENTITY")...</Alex>
+def _get_column_semantic_type_name(validator, column) -> SemanticDomainTypes:
```

Comment:
Explanation: James and I used identity as the default domain for an Expectation as we could not assume column expectation or table expectation as domains - it is meant to be a default

Reply:
@gilpasternak35 Thank you so much for this explanation!  This really helps.  (Though we can probably accomplish this in a simpler, more formal way -- perhaps in a future PR.)

Reply:
@gilpasternak35 Based on conversation with @anthonyburdi the usefulness of this method became clear; for now, I will clean it up and add comments for the future.  Thank you.